core/math/bsp_tree.cpp

```diff
@@ -577,12 +577,11 @@ BSP_Tree::BSP_Tree(const PoolVector<Face3> &p_faces, real_t p_error_radius) {
 	error_radius = p_error_radius;
 }
 
-BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius) {
-
-	nodes = p_nodes;
-	planes = p_planes;
-	aabb = p_aabb;
-	error_radius = p_error_radius;
+BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius)
```

Comment:
I understand, I don't like it either. I applied `clang-format` to your code block, but unfortunately `clang-format` reverts it back to the original code in this pull request and keeps putting tabs and spaces at the beginning of the line. This is other code block was also formatted with `clang-format` but the tabs and spaces are still there:

```cpp
BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius)
	:

	  nodes(p_nodes),
	  planes(p_planes),
	  aabb(p_aabb),
	  error_radius(p_error_radius) {
}
```

I'm still a bit new to `clang-format` but as far as I can tell it can be configured somewhat and seems like we should be able to tweak this behavior. I'll check it's configuration options to see if there's anything we can do here.